### PR TITLE
Migration to Version 2.0.0

### PR DESCRIPTION
**Pull request by FarhanMS123**
Hi,

As my promise (even though it already been months), I done this documentation for Vite Multiple Assets migration to Version 2,0.0. The original code can be look in this PR: https://github.com/nguyenbatranvan/vite-multiple-assets/pull/16 .

I also modify in how options documented, and giving examples and tricks in each options based on PR.

Thanks,

### CODE REVIEW

**Comment by nguyenbatranvan** [comment]
> Hi,
> 
> As my promise (even though it already been months), I done this documentation for Vite Multiple Assets migration to Version 2,0.0. The original code can be look in this PR: [nguyenbatranvan/vite-multiple-assets#16](https://github.com/nguyenbatranvan/vite-multiple-assets/pull/16) .
> 
> I also modify in how options documented, and giving examples and tricks in each options based on PR.
> 
> Thanks,

in the docs, is there any guidance for the output directory not to include the parent directory?
For example: `repo/assets`. I only need to get all the content in assets and not need to get `assets`. Or vice versa get all the content including the root `assets`?

File: docs/guide/index.md [hunk 1]
```diff
@@ -17,16 +17,149 @@ export default defineConfig({
...
+```ts
+var assets: IAssets = [];
+type IAssets = string[];
+```
+
+List copied `assets` by following pattern defined in [`fast-glob` pattern](https://github.com/mrmlnc/fast-glob?tab=readme-ov-file#pattern-syntax) which is [Unix Glob](https://man7.org/linux/man-pages/man7/glob.7.html) [Pattern](https://www.gnu.org/software/bash/manual/html_node/Pattern-Matching.html). There are few things to take:
+
+- always use forward-slash `/` for path, and use backslash to escape character
+- do not use single-dot `./**/*.txt` `./abc/**/*.txt` for current path, instead straight use the pattern `**/*.txt` `abc/**/*.txt`
+- If you are hope certain folder included to be copied, make it in selectable list pattern, `dir1/{\x01,folder_name}/**`. Just ensure there is no folder or file named by first ascii
```
Comment (FarhanMS123) [inline docs/guide/index.md:111]: I don't think so. But, I believe this section could works?

Comment (nguyenbatranvan) [reply docs/guide/index.md:111]: only `dir1/folder_name/**` will ignore `folder_name`, i tried it and it worked

Comment (FarhanMS123) [reply docs/guide/index.md:111]: Yes. It is expected to have that behavior. The `dir1/folder_name/**` would copy all content inside it except the folder itself. If you want to include the folder, you could make the folder to be a glob pattern, e.g. `dir1/{\x01,folder_name}/**` (Line 111).

Maybe, I am not too clear in defining this tricks and behavior inside the docs.

Comment (FarhanMS123) [reply docs/guide/index.md:111]: > I don't think so. But, I believe this section could works?

Oh. I mean by "works" as "works to be a guidance for user".

**Pull request merged**